test/createAzureResource.test.ts

```diff
@@ -6,30 +6,30 @@
 import * as assert from 'assert';
 import { ResourceManagementClient } from 'azure-arm-resource';
 import { WebSiteManagementClient, WebSiteManagementModels } from 'azure-arm-website';
+import { execSync } from 'child_process';
+import * as fse from 'fs-extra';
 import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as path from 'path';
 import * as vscode from 'vscode';
-import { AzExtTreeDataProvider, AzureAccountTreeItemWithProjects, DialogResponses, ext, getGlobalSetting, getRandomHexString, ProjectLanguage, projectLanguageSetting, TestAzureAccount, TestUserInput, updateGlobalSetting } from '../extension.bundle';
+import { AzExtTreeDataProvider, AzureAccountTreeItemWithProjects, delay, DialogResponses, ext, FunctionTreeItem, getRandomHexString, ProjectLanguage, ProjectRuntime, TestAzureAccount, TestUserInput } from '../extension.bundle';
 import { longRunningTestsEnabled } from './global.test';
 import { runWithFuncSetting } from './runWithSetting';
+import { getCSharpValidateOptions, getJavaScriptValidateOptions, validateProject } from './validateProject';
 
 // tslint:disable-next-line:max-func-body-length
 suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Promise<void> {
     this.timeout(1200 * 1000);
     const resourceGroupsToDelete: string[] = [];
     const testAccount: TestAzureAccount = new TestAzureAccount();
-    let oldProjectLanguage: ProjectLanguage | undefined;
     let webSiteClient: WebSiteManagementClient;
     const resourceName1: string = getRandomHexString().toLowerCase();
+    const projectPath: string = path.join('D:', 'testOutput');
```

Comment:
This line is causing problems. While this will work for Windows, the D drive doesn't work with Linux and Mac.

![image](https://user-images.githubusercontent.com/5290572/57961803-2d470e80-78c6-11e9-96ed-70216f35893c.png)

I think you could use the `os` library for this.

`import * as os from 'os';`
And then call `os.tmpdir();`

There might be a better command, feel free to investigate 😃